inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/source/MySQLBinlogSource.java

```diff
@@ -100,4 +100,10 @@ public class MySQLBinlogSource extends StreamSource {
 
     @ApiModelProperty(value = "Primary key must be shared by all tables", required = false)
     private String primaryKey;
+
+    @ApiModelProperty("Directly read binlog from the specified offset filename")
+    public String specificOffsetFile;
```

Comment:
Maybe change to the `offsetFile` and `offsetPosition` are more readable.

Reply:
Referring to the names used by Flink CDC are specificOffsetFile and specificOffsetPos, and the agent module also uses this name.
![image](https://user-images.githubusercontent.com/20356765/169936098-71b02658-1c31-4b63-abb2-09e7c5f9f25f.png)